src/index.js

```diff
@@ -1,6 +1,8 @@
 export {default as WebexAdapter} from './WebexAdapter';
 export {default as ActivitiesAdapter} from './ActivitiesAdapter';
-export {default as MeetingsAdapter, MeetingControlState, MeetingState} from './MeetingsAdapter';
+export {
+  default as MeetingsAdapter, MeetingControlState, MeetingState, MediaPermissions,
```

Comment:
can you have each export in their own line?
```suggestion
  default as MeetingsAdapter,
  MeetingControlState,
  MeetingState,
  MediaPermissions,
```

Reply:
Done.